File: PRAK301-2210817220011-MeldianaHandayani.py
A= int(input("Bilangan 1 = "))
B= int(input("Bilangan 2 = "))
C= int(input("Bilangan 3 = "))
if A<B and B<C:
     print(A,B,C)
elif A<C and C<B:
    print(A,C,B)
elif B<A and A<C:
    print(B,A,C)
elif B<C and C<A:
    print (B,C,A)
elif C<A and A<B:
    print(C,A,B)
if  C<B and B<A:
    print(C,B,A)
    
    print("urutan bilangan terkecil adalah ",  A, B, C)

File: PRAK301-2210817220011-MeldianaHandayani.c
#include <stdio.h>
 int main ()
 {
  int A, B, C, n1, n2, n3;
printf("Bilangan 1 = ");
    scanf("%d", &A);
printf("Bilangan 2 = ");
    scanf("%d", &B);
printf("Bilangan 3 = ");
    scanf("%d", &C);
if (A<B && B<C){
    n1=A; n2=B; n3=C;
}
else if(A<C && C<B){
    n1=A; n2=C; n3=B;
}
else if(B<A && A<C){
     n1= B; n2=A; n3=C;
}
else if (B<C && C<A){
    n1=B; n2=C; n3=A;
}
else if(C<A && A<B){
    n1= C; n2=A; n3=B;
}
else if (C<B && B<A){
    n1=C; n2=B; n3=A;  
}
printf(" %d %d %d ",  n1, n2, n3);
}